cockatrice/src/SpoilerBackgroundUpdaterThread.cpp

```diff
@@ -0,0 +1,249 @@
+#include <QDateTime>
```

Comment:
If you are working on a Mac, the filesystem is case insensitive, so it won't pick up changes to the case of file names.. you can probably workaround it by renaming the file to something else, committing, then rename again to the correct name and re-commit.